libs/langchain/langchain/tools/eleven_labs/text2speech.py

```diff
@@ -0,0 +1,67 @@
+import tempfile
+from typing import Dict, Union
+
+from langchain.pydantic_v1 import root_validator
+from langchain.tools.base import BaseTool
+from langchain.tools.eleven_labs.models import ElevenLabsModel
+from langchain.utils import get_from_dict_or_env
+
+try:
+    import elevenlabs
+
+except ImportError:
+    raise ImportError(
+        "elevenlabs is not installed. " "Run `pip install elevenlabs` to install."
+    )
+
+
+class ElevenLabsText2SpeechTool(BaseTool):
+    """Tool that queries the Eleven Labs Text2Speech API.
+
+    In order to set this up, follow instructions at:
+    https://docs.elevenlabs.io/welcome/introduction
+    """
+
+    model: Union[ElevenLabsModel, str] = ElevenLabsModel.MULTI_LINGUAL
+
+    name: str = "eleven_labs_text2speech"
```

Comment:
from tools(ie LLMs) perspective does it matter that it's eleven labs? same with mentioning eleven labs in description

Reply:
it is analogous to: [azure text-to-speech](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/tools/azure_cognitive_services/text2speech.py
) component. Implemented this way to keep consistency